### PR TITLE
feat: Support for Series/Parent items on Web Embeds

### PR DESCRIPTION
**Pull request by vinnyjth**
## 🐛 Issue

Liquid wants to show series data in web embeds. 

https://3.basecamp.com/3926363/buckets/32694519/card_tables/cards/6966844981

## ✏️ Solution

Render a hero-like car showing series data, if there's a series attached.

## 🔬 To Test

1. Start the `micro-service` project
2. [Visit this item! ](http://localhost:3000/?id=psalm-8-VW5pdmVyc2FsQ29udGVudEl0ZW0tNDMwNDFmZmEtNzI1OC00OTFkLTk3ZTQtNzVmNDkyOTFmYmY0)

## 📸 Screenshots

CleanShot 2024-01-25 at 20 07 17@2x
CleanShot 2024-01-25 at 20 06 50@2x

### CODE REVIEW

**Comment by vinnyjth** [comment]
@conrad-vanl I would appreciate any design feedback if you have time! feel free to edit my code if it's easier.

**Review by nlewis84 — APPROVED** [review]
Looks like it matches the wireframe to me. This is just for the parent series data, right? It looks like sibling content is just getting stacked below and we would probably want that in a carousel (but maybe this is something to solve in another PR).

File: packages/web-shared/components/ContentSingle.js [hunk 1]
```diff
@@ -255,6 +257,54 @@ function ContentSingle(props = {}) {
           </Box>
         ) : null}
         {/* Display content for sermons */}
+        {hasParent ? (
+          <>
+            <H3 flex="1" mr="xs" mb="xs">
+              This Series
+            </H3>
+            <Box flex="1" mb="l" minWidth="180px" {...props}>
```
Comment (conrad-vanl) [inline packages/web-shared/components/ContentSingle.js:265]: I assume there's not existing card components?

Comment (vinnyjth) [reply packages/web-shared/components/ContentSingle.js:265]: There's not, and there was enough difference between this card and the hero card that I punted on trying to DRY it up

**Pull request merged**